src/main/scala/is/hail/expr/Type.scala

```diff
@@ -1239,6 +1239,24 @@ object TGenotype {
       "pl" -> TArray(TInt32()))
     if (required) (!t).asInstanceOf[TStruct] else t
   }
+  
+  def representationWithVCFAttributes(required: Boolean = false): TStruct = {
```

Comment:
I'm not entirely sure I like this here, since the Type classes are already pretty bloated, but I guess I don't really mind if it stays here until we overhaul the structure of this file.

Reply:
This will go away when TGenotype goes away.